Lib/uuid.py

```diff
@@ -799,5 +807,10 @@ def main():
 NAMESPACE_OID = UUID('6ba7b812-9dad-11d1-80b4-00c04fd430c8')
 NAMESPACE_X500 = UUID('6ba7b814-9dad-11d1-80b4-00c04fd430c8')
 
+# RFC 9562 Sections 5.9 and 5.10 define the special Nil and Max UUID formats.
+
+NIL = UUID('00000000-0000-0000-0000-000000000000')
+MAX = UUID('ffffffff-ffff-ffff-ffff-ffffffffffff')
```

Comment:
```suggestion
NIL = UUID(int=0)
MAX = UUID(int=(1 << 128) - 1)
```

Can you time the import time of `uuid` if you switch to an `int` argument? the reason is that the hexadecimal form requires to pre-process the input a bit more and since we're already optimizing the import time of `uuid` quite a lot (just look at the commit history), I think it's worth to check this.

Reply:
So I'd originally started off using `UUID(int=0)`, but settled on the string form as that is how it is explicitly displayed in the RFC (although it does also mention all zero/one bits). In addition, the other constants in the module, e.g. `NAMESPACE_OID`, etc. also use the string form.

There's not much point timing the import of `uuid` - easier to just compare the string parsing vs using `int=`:

```pycon
In [1]: from uuid import UUID

In [2]: %timeit UUID('00000000-0000-0000-0000-000000000000')
1.2 μs ± 48.6 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [3]: %timeit UUID(int=0)
910 ns ± 48.9 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [4]: %timeit UUID('ffffffff-ffff-ffff-ffff-ffffffffffff')
1.21 μs ± 41.1 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [5]: %timeit UUID(int=2 ** 128 - 1)
1.19 μs ± 53.5 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [6]: %timeit UUID(int=(1 << 128) - 1)
1.01 μs ± 62.5 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```

_(Note that this was using Python 3.13.1.)_

Using `int=` is a little faster, but it's so minute a difference I'm not sure it's worth it.

Reply:
No indeed, not worth the change. (even with the optimized UUID constructor I've provided in #128151, I doubt it will change much)